src/samplers/orthogonal.cpp

```diff
@@ -0,0 +1,225 @@
+#include <mitsuba/core/profiler.h>
+#include <mitsuba/core/properties.h>
+#include <mitsuba/core/spectrum.h>
+#include <mitsuba/render/sampler.h>
+
+NAMESPACE_BEGIN(mitsuba)
+
+/**!
+
+.. _sampler-orthogonal:
+
+Orthogonal Array sampler (:monosp:`orthogonal`)
+-----------------------------------------------
+
+.. pluginparameters::
+
+ * - sample_count
+   - |int|
+   - Number of samples per pixel. This value has to be the square of a prime number. (Default: 4)
+ * - strength
+   - |int|
+   - Orthogonal array's strength (Default: 2)
+ * - seed
+   - |int|
+   - Seed offset (Default: 0)
+ * - jitter
+   - |bool|
+   - Adds additional random jitter withing the substratum (Default: True)
+
+This plugin implements the Orthogonal Array sampler generator introduced by
+:cite:`jarosz19orthogonal`. It generalizes correlated multi-jittered sampling to higher dimensions
+by using *orthogonal arrays (OAs)* of strength :math:`s`. The strength property of OAs tells that
+projecting the generated samples to any combination of :math:`s` dimensions will always result in
+a well stratified pattern. In other words, when :math:`s=2` (default value), the high-dimentional
+samples are simultaneously stratified in all 2D projections as if they had been produced by
+correlated multi-jittered sampling. By construction, samples produced by this generator are also
+well stratified when projected on both 1D axis.
```

Comment:
Why wouldn't one choose `s=1000` here to achieve the best stratification over many dimensions? Needs a discussion of the pros/cons.

Reply:
This isn't how it works. That is the difference between Orthogonal Arrays and Strong Orthogonal Arrays. For OA, the stratification will be good only for the s-tuple of dimensions and for the 1D projections. E.g. when `s=3`, there is no garantee that stratification hold in 2D.

Maybe this wasn't clear in the comment, I will try to add a few words on this.